src/components/network/line-layer.js

```diff
@@ -64,34 +65,57 @@ export const LineLoadingZone = {
 };
 
 export function getLineLoadingZoneOfSide(
```

Comment:
i suggest to optimise the code of this function to be like this: 
```
if (limit === undefined || (intensity === undefined && activeFlow === undefined) || (intensity === 0 && activeFlow === 0)) {
    return LineLoadingZone.UNKNOWN;
} 
const valIntensity = intensity !== undefined ? intensity : activeFlow * 1000 / (voltagLevel.nominalVoltage * Math.sqrt(3) * DEFAULT_COSPHI);
const threshold = (lineFlowAlertThreshold * limit) / 100;
return valIntensity > 0 && valIntensity < threshold ? LineLoadingZone.SAFE : valIntensity >= threshold && valIntensity < limit ? LineLoadingZone.WARNING : LineLoadingZone.OVERLOAD;
```

- you can remove the first if statement and combine the conditions to check for undefined or zero values into a single ternary operator.

- instead of checking for intensity !== undefined, you can use the ternary operator to assign a default value of (activeFlow * 1000) / (voltagLevel.nominalVoltage * Math.sqrt(3) * DEFAULT_COSPHI) when intensity is undefined.

- you can use a single return statement with a ternary operator to determine the line loading zone based on the value of valIntensity.